Cargo.toml

```diff
@@ -26,6 +26,7 @@ nix = { version = "0.28", features = ["fs"] }
 predicates = "3"
 serial_test = "3.0"
 tempfile = "3"
+pretty_assertions = "0.6.1"
```

Comment:
Do you use this version on purpose? It's quite old, the current version is `1.4.0`.

Reply:
bad copy and paste, fixed :)